frontend/resources/translations/en.json

```diff
@@ -678,7 +684,8 @@
       "median": "Median",
       "min": "Min",
       "sum": "Sum"
-    }
+    },
+    "removeSorting": "Would you like to remove sorting?"
```

Comment:
Maybe move this to `grid` > `sort` so that it lives with the other tokens there?